collections-csharp/MergingDictionariesInCSharp/MergingDictionariesInCSharp.Tests/UnitTests.cs

```diff
@@ -0,0 +1,109 @@
+using static MergingDictionariesInCSharp.Methods;
+
+[TestFixture]
+public class MergingMethodsTests
+{
+    private Dictionary<int, string> _dictionaryA;
+    private Dictionary<int, string> _dictionaryB;
+    private Dictionary<int, string> _expectedMergedDictionary;
+
+    [SetUp]
+    public void SetUp()
+    {
+        _dictionaryA = new Dictionary<int, string>()
+        {
+            { 1, "Apple" },
+            { 2, "Banana" },
+            { 3, "Orange" }
+        };
+
+        _dictionaryB = new Dictionary<int, string>()
+        {
+            { 4, "Grapes" },
+            { 5, "Watermelon" },
+            { 6, "Pineapple" }
+        };
+
+        _expectedMergedDictionary = new Dictionary<int, string>()
+        {
+            { 1, "Apple" },
+            { 2, "Banana" },
+            { 3, "Orange" },
+            { 4, "Grapes" },
+            { 5, "Watermelon" },
+            { 6, "Pineapple" }
+        };
+    }
+
+    [Test]
+    public void ConcatMethod_ShouldMergeDictionaries()
```

Comment:
These should all be `Given_When_Then`.

Reply:
Changed the names here in every test method